codecov.yaml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 79%
+    patch:
+      default:
+        target: 100%
+notify:
+  after_n_builds: 12
```

Comment:
This has codecov wait until it receives 12 report files before reporting the coverage back. If I counted correctly this should be the total number of reports that get uploaded.

Without this everything works, but can be confusing because coverage is sent back with each file received. This results in low, failing numbers slowly increasing until all files are received.

Reply:
Decided to change this to 4. There are 4 tests run for 3 different python version resulting in 12 total. So I think we can report at 4 and have a full picture. If this turns out to be wrong we can adjust.